kevm-pyk/src/kontrol/foundry.py

```diff
@@ -413,10 +416,13 @@ def help_info() -> list[str]:
         return res_lines
 
     def proofs_with_test(self, test: str) -> list[Proof]:
+        #          print([pid.split(':')[0] for pid in listdir(self.proofs_dir)])
+        #          print(single(self._escape_brackets([test])))
```

Comment:
Make sure to remove these debug comments for the final version.

Reply:
Done.